test/integration/eslint/custom-config/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "eslint-custom-config",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "devDependencies": {
+    "eslint-config-next": "file:../../../../packages/eslint-config-next",
```

Comment:
Will update to use the npm version when plugin is published to [`eslint-config-next`](https://www.npmjs.com/package/eslint-config-next)

Reply:
Because yarn workspaces are used in development `node_modules/eslint-config-next` in the root should already be linked

Reply:
Ah fixed 👍